usaspending_api/common/api_request_utils.py

```diff
@@ -464,12 +466,16 @@ def build_response(self):
                     "matched_ids": Location.objects.filter(Q(**{"location_congressional_code": row[0], "location_state_code": row[1], "location_state_name": row[2]})).values_list("location_id", flat=True)
                 }
                 response_object.append(response_row)
+                if len(response_object) >= limit:
```

Comment:
Thanks for poking around at this. Fair point that this isn't a model-based endpoint, and I def. don't think we need to make a hierarchical geography model. What I would like to do--and we don't need to do this now--is to migrate the functions in `api_request_utils.py` to a place consistent with DRF (common.mixins, etc).